### ISSUE TITLE
Feature: Make Demo Data as Optional

### ISSUE THREAD

**Issue by NeftaliYagua** [innocommerce/innoshop#7]
Considering making demo data options would be best for future releases.

**Comment by yushine** [comment]
Yes, you are correct. During the web installation, an option can be provided; if it is not checked, only necessary data is imported, such as settings, locale, and currency, without importing data like categories, products, and brands.
